tests/integration-tests/tests/schedulers/test_custom_munge_key.py

```diff
@@ -48,24 +49,35 @@ def test_custom_munge_key(
     7. Roll back with failure: Update cluster to add back the custom munge key. But let the cluster update fail after
        the custom munge key has been added. Trigger cluster roll back. Test if munge key is fully functional.
     """
-    encode_custom_munge_key = create_base64_encoded_key()
+    encoded_custom_munge_key = create_base64_encoded_key()
     custom_munge_key_arn = store_secret_in_secret_manager(
         region,
-        secret_string=encode_custom_munge_key,
+        secret_string=encoded_custom_munge_key,
     )
     cluster_config = pcluster_config_reader(custom_munge_key_arn=custom_munge_key_arn)
     cluster = clusters_factory(cluster_config, upper_case_cluster_name=True)
 
+    # 1. Deployment verification: Confirm munge key is successfully shared across head, compute and login nodes,
+    # and jobs can be submitted.
+
     # Test if the munge key was successfully fetched, decoded and shared in HeadNode and LoginNodes
     remote_command_executor = RemoteCommandExecutor(cluster)
-    _test_custom_munge_key_fetch_and_decode(remote_command_executor, encode_custom_munge_key)
+    check_encoded_munge_key(remote_command_executor, encoded_custom_munge_key)
     _test_munge_key_shared(remote_command_executor)
 
     remote_command_executor_login = RemoteCommandExecutor(cluster, use_login_node=True)
-    _test_custom_munge_key_fetch_and_decode(remote_command_executor_login, encode_custom_munge_key)
+    check_encoded_munge_key(remote_command_executor_login, encoded_custom_munge_key)
+
+    # Test if compute node can run jobs from LoginNodes and HeadNode, indicating the munge key was successfully fetched.
+    scheduler_commands = scheduler_commands_factory(remote_command_executor_login)
+    scheduler_commands.submit_command_and_assert_job_accepted(
+        submit_command_args={
+            "command": "srun sleep 1",
+            "nodes": 2,
+        }
+    )
     remote_command_executor_login.close_connection()
 
-    # Test if compute node can run jobs, indicating the munge key was successfully fetched.
     scheduler_commands = scheduler_commands_factory(remote_command_executor)
     scheduler_commands.submit_command_and_assert_job_accepted(
```

Comment:
Sorry, I just noticed this. Please use [`submit_command_and_assert_job_succeeded`](https://github.com/aws/aws-parallelcluster/blob/cf8b19fc1b5f465b4914ab95414b5099f38e47d8/tests/integration-tests/tests/common/schedulers_common.py#L538) here instead of `submit_command_and_assert_job_accepted`. This way we verify that the job actually worked.

Reply:
Done!

Reply:
Also done for another one. In the roll back test.